src/utils/patreonApi.js

```diff
@@ -0,0 +1,13 @@
+import { request } from "./api";
+
+const getPatreonPosts = () => {
+  return request("/api/patreon")
+    .then((posts) => {
+      return posts;
+    })
+    .catch((err) => {
```

Comment:
You should **not** use `catch` in `Api` because you need to `catch` possible errors at the very **end** of any request, and this end is in (`App.js`) after all `then` blocks. And in real projects the `catch` blocks usually open error popups and so on.